lib/localenv/credentials/credentials.go

```diff
@@ -75,6 +75,18 @@ func credentialsFromEntry(entry users.LoginEntry) *Credentials {
 	}
 }
 
+// FromTokenAndHub creates new credentials from the provided token and address.
+func FromTokenAndHub(token, hub string) *Credentials {
```

Comment:
Will this work if only one of the parameters set?

Reply:
Yeah, you can for example set only `--hub` and it will search for its credentials in regular key stores. You can't only set `--token` though - the CLI will ask to set `--hub` as well.